src/uTensor/ops/NnOps.hpp

```diff
@@ -33,17 +33,32 @@ void Softmax(S_TENSOR input, S_TENSOR output)
     Tout* out_ptr = output->write<Tout>(0,0);
     const Tin* in_ptr = input->read<Tin>(0, 0);
 
-    for (size_t row_dim = 0; row_dim < row_dim_max; ++row_dim){
+    for (size_t row_dim = 0; row_dim < row_dim_max; ++row_dim) {
         size_t base_offset = row_dim * row_stride;
+
+        Tout max = 0;
```

Comment:
I'm using `0` here instead of `-INFINITY` because this is only an issue with values above 0. Also with 0 it will work with integer types too.

Reply:
I see.
Just see my comment below and merge the base branch (not `fix-softmax`).
You should pass the CI afterward.